_data-prepper/pipelines/configuration/processors/grok.md

```diff
@@ -53,4 +70,4 @@ The `Grok` processor includes the following custom metrics.
 
 ### Timer
 
-* `grokProcessingTime`: The time taken by individual records to match against patterns from `match`. The `avg` metric is the most useful metric for this timer because it provides you with an average value of the time it takes records to match. 
\ No newline at end of file
+* `grokProcessingTime`: The time taken by individual records to match against patterns from `match`. The `avg` metric is the most useful metric for this timer because it provides the average value of the time it takes records to match.
```

Comment:
```suggestion
* `grokProcessingTime`: The time taken by individual records to match against `match` patterns. The `avg` metric is the most useful metric for this timer because it provides the average value of the time it takes records to match.
```